package/MDAnalysis/core/selection.py

```diff
@@ -260,6 +260,10 @@ def apply(self, group):
         sel = self.sel.apply(group)
         # All atoms in group that aren't in sel
         sys = group[~np.in1d(group.indices, sel.indices)]
+        # all "around" selections with a zero
+        # distance will produce an empty atomgroup
+        if self.cutoff == 0.0:
+            return sys[0:0]
```

Comment:
Just being pedantic here, but what if I have two atoms superposed? Is there an easy solution for this case?

Reply:
I'll see what I can do. It isn't a case in our testsuite though, so we're not explicitly supporting it yet.

Reply:
Ok, I've revised with code + unit test for the scenario. It can be a little mind bending because of self-exclusion rules.